tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -41,3 +44,34 @@ def users_to_deactivate(days):
         last_login__gte=datetime.now(tz=timezone.utc) - timedelta(days=days+1),
         account_approval_status=AccountApprovalStatusChoices.APPROVED,
         )
+
+def get_ofa_admin_user_emails():
+    """Return a list of OFA System Admin users."""
+    return [user.email for user in User.objects.filter(
+        groups='4'
+    )]
+
+def get_num_access_requests():
+    """Return the number of users requesting access."""
+    return len(User.objects.filter(
+        account_approval_status=AccountApprovalStatusChoices.ACCESS_REQUEST,
+    ))
+
+@shared_task
+def email_admin_num_access_requests():
+    """Send all OFA System Admins an email with how many users have requested access."""
+    recipient_email = get_ofa_admin_user_emails()
+    text_message = ''
+    subject = 'Number of Active Access Requests'
+    url = settings.FRONTEND_BASE_URL + reverse("admin:users_user_changelist")
```

Comment:
just icing on the cake, but could this `url` implement the query params for filtering and ordering as well? we could link directly to the list of users in `Access Request` state, ordered by the date/time requested